### ISSUE TITLE
libncursesw.so.6: no version information available

### ISSUE THREAD

**Issue by muralikodali** [simonmichael/hledger#1748]
with recent update on Arch Linux, i am getting the following warning:

```
hledger: /usr/lib/libncursesw.so.6: no version information available (required by /usr/lib/libHShledger-1.22.2 ET5Hl5xVjiU1OND4wgZagF-ghc9.0.1.so)
```

**Comment by simonmichael** [comment]
Thanks for the report. Searching for the error message suggests that installing `ncurses5-compat-libs` might help ? Let us know.

**Comment by muralikodali** [comment]
Thanks for your suggestion.

installing ncurses5-compat-libs not helped to remove the warning.

i have upgraded my system again, which included upgrading ncurses to ncurses-6.3-1.
now the error is gone.
